amun/base/imagestream.yaml

```diff
@@ -22,3 +22,19 @@ metadata:
 spec:
   lookupPolicy:
     local: true
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: workflow-helpers
+spec:
+  lookupPolicy:
+    local: true
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: messaging
+spec:
+  lookupPolicy:
+    local: true
```

Comment:
you also need to create one `imagestreamtag.yaml` in the different overlays and reference them in the `kustomize.yaml` file in respective overlays, otherwise no specific image is used. for example: https://github.com/thoth-station/thoth-application/blob/master/amun/overlays/ocp4-stage/amun-api/imagestreamtag.yaml

Reply:
This is only necessary in `prod`, `stage`, and `ocp4-stage`? (any that already have `imagestreamtag.yaml` defined in the overlay)

Reply:
Nevermind, I realize that `power9` is the only overlay with an `imagestreamtag.yaml`